### ISSUE TITLE
The model is not trainable

### ISSUE THREAD

**Issue by kingman1980** [orobix/retina-unet#71]
Keras: 2.2.4
Tensorflow: 1.14.0

During training, the loss became smaller.
But when I use the best model to test, there is nothing in the predicted image.

Epoch 00001: val_loss improved from inf to 0.32606, saving model to 
Epoch 2/150
 - 49s - loss: 0.3457 - acc: 0.8952 - val_loss: 0.3242 - val_acc: 0.9042

Epoch 00002: val_loss improved from 0.32606 to 0.32418, saving model to 
Epoch 3/150
 - 54s - loss: 0.3440 - acc: 0.8952 - val_loss: 0.3230 - val_acc: 0.9042

Epoch 00003: val_loss improved from 0.32418 to 0.32298, saving model to 
Epoch 4/150
 - 56s - loss: 0.3429 - acc: 0.8952 - val_loss: 0.3221 - val_acc: 0.9042

Epoch 00004: val_loss improved from 0.32298 to 0.32206, saving model to 
Epoch 5/150
 - 55s - loss: 0.3420 - acc: 0.8952 - val_loss: 0.3212 - val_acc: 0.9042

Epoch 00005: val_loss improved from 0.32206 to 0.32117, saving model to 

someone can tell me why?
